source/matplot/core/axis_type.cpp

```diff
@@ -257,7 +263,13 @@ namespace matplot {
                     char buff[20] = {0};
                     time_t now = (time_t)tick_values_[i];
                     struct tm buf;
+#if defined(__unix__)
+                    localtime_r(&now, &buf);
+#elif defined(_WIN32)
                     localtime_s(&buf, &now);
+#else
+                    buf = std::localtime(&now);
```

Comment:
We have a bug here:

https://github.com/alandefreitas/matplotplusplus/runs/7770952743?check_suite_focus=true#step:5:20